algo/src/main/java/org/neo4j/graphalgo/PageRankProc.java

```diff
@@ -177,10 +186,14 @@ private PageRankResult evaluate(
         final int concurrency = configuration.getConcurrency(Pools.getNoThreadsInDefaultPool());
         log.debug("Computing page rank with damping of " + dampingFactor + " and " + iterations + " iterations.");
 
+
+        List<Node> sourceNodes = configuration.get("sourceNodes", new ArrayList<>());
+        Stream<Long> sourceNodeIds = sourceNodes.stream().map(Node::getId);
```

Comment:
We could use `mapToLong` to avoid boxing to Long instances

Reply:
Have updated this now